src/Platform/Microsoft.Testing.Platform/ServerMode/Json/Json.cs

```diff
@@ -155,6 +746,23 @@ internal T Bind<T>(JsonElement element, string? property = null)
             }
         }
 
+        return Deserialize<T>(element);
+    }
+
+    internal bool TryBind<T>(JsonElement element, out T? value, string? property = null)
+    {
+        if (property != null && !element.TryGetProperty(property, out element))
```

Comment:
```suggestion
        if (property is not null && !element.TryGetProperty(property, out element))
```

Use the `is` and `is not` operator for nullable check `==` and `!=` works 99.9% of cases, we can sync on it.

Reply:
Sure.